### PR TITLE
added hotfix to show plan in heroku and show in navbar

### PR DESCRIPTION
**Pull request by curtydudes**
### THE STORY
-added hotfix in webhooks file that will show what plan was availed (month or year)
-added feature that will show different nav bar links when user is a paid user or a free user
-added RSPEC tests for calling the api and webhook

### SCREENSHOTS
image

#### AFTER MERGE DO
```
bundle install
```

### CODE REVIEW

File: app/controllers/static_pages_controller.rb [hunk 1]
```diff
@@ -1,5 +1,9 @@
 class StaticPagesController < ApplicationController
-  def home_page; end
+  def home_page
+    @users = User.all
+    @free_user = User.where(status: 0)
+    @paid_user = User.where(status: 1)
```
Comment (analyn-cajocson) [inline app/controllers/static_pages_controller.rb:5]:
```suggestion
    @free_user = @users.where(status: 0)
    @paid_user = @users.where(status: 1)
```

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

**Review by mumble-07 — APPROVED** [review]
lgtm!

File: app/controllers/static_pages_controller.rb (hunk 1 above)
Comment (curtydudes) [reply app/controllers/static_pages_controller.rb]: got this. thanks maam!

**Review by bryanarboledaa — APPROVED** [review]
LGTM

**Review by analyn-cajocson — APPROVED** [review]

**Pull request merged**